examples/medplum-patient-intake-demo/data/core/patient-intake-questionnaire.json

```diff
@@ -186,40 +192,9 @@
             "type": "group",
             "item": [
               {
-                "linkId": "acknowledgement-for-advance-directives-choice",
-                "text": "An Advance Medical Directive is a document by which a person makes provision for health care decisions in the event that, in the future, he/she becomes unable to make those decisions.\nPlease select one option below:",
-                "type": "choice",
-                "answerOption": [
-                  {
-                    "valueCoding": {
-                      "system": "http://terminology.hl7.org/CodeSystem/v2-0532",
-                      "code": "Y",
-                      "display": "Yes, I do have an Advance Directive / Living Will / Durable Power of Attorney for medical or health care decisions."
-                    }
-                  },
-                  {
-                    "valueCoding": {
-                      "system": "http://terminology.hl7.org/CodeSystem/v2-0532",
-                      "code": "N",
-                      "display": "No, I do NOT have an Advance Directive / Living Will / Durable Power of Attorney for medical or health care decisions."
-                    }
-                  }
-                ]
-              },
-              {
-                "linkId": "acknowledgement-for-advance-directives-email",
-                "text": "If you do have an Advance Directive, please make sure to send a copy to us, in person or by mail.",
-                "type": "boolean",
-                "enableWhen": [
-                  {
-                    "question": "acknowledgement-for-advance-directives-choice",
-                    "operator": "=",
-                    "answerCoding": {
-                      "system": "http://terminology.hl7.org/CodeSystem/v2-0532",
-                      "code": "Y"
-                    }
-                  }
-                ]
+                "linkId": "acknowledgement-for-advance-directives-help",
```

Comment:
Why this change?

Reply:
We made those changes to simplify the modeling. As we couldn't find how to model those consent-related information, we'd need to dive deeper into that.